src/Sentry.Maui/Internal/MauiGestureRecognizerEventsBinder.cs

```diff
@@ -0,0 +1,197 @@
+                drag.DragStarting -= OnDragStartingGesture;
+                drag.DropCompleted -= OnDropCompletedGesture;
+
+                if (bind)
+                {
+                    drag.DragStarting += OnDragStartingGesture;
+                    drag.DropCompleted += OnDropCompletedGesture;
+                }
+                break;
+
+            case PanGestureRecognizer pan:
+                pan.PanUpdated -= OnPanGesture;
+
+                if (bind)
+                {
+                    pan.PanUpdated += OnPanGesture;
+                }
+                break;
+
+            case PointerGestureRecognizer pointer:
+                pointer.PointerEntered -= OnPointerEnteredGesture;
+                pointer.PointerExited -= OnPointerExitedGesture;
+                pointer.PointerMoved -= OnPointerMovedGesture;
+                pointer.PointerPressed -= OnPointerPressedGesture;
+                pointer.PointerReleased -= OnPointerReleasedGesture;
+
+                if (bind)
+                {
+                    pointer.PointerEntered += OnPointerEnteredGesture;
+                    pointer.PointerExited += OnPointerExitedGesture;
+                    pointer.PointerMoved += OnPointerMovedGesture;
+                    pointer.PointerPressed += OnPointerPressedGesture;
+                    pointer.PointerReleased += OnPointerReleasedGesture;
+                }
+                break;
+        }
+    }
+
+    private static void OnPointerReleasedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerReleased),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerPressedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerPressed),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerMovedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerMoved),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerExitedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerExited),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerEnteredGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerEntered),
+        ToPointerData(e)
+    ));
+
+    private static IEnumerable<(string Key, string Value)> ToPointerData(PointerEventArgs e) =>
+    [
+        // some of the data here may have some challenges being pulled out
+        #if ANDROID
+        ("MotionEventAction", e.PlatformArgs?.MotionEvent.Action.ToString() ?? string.Empty)
+        //("MotionEventActionButton", e.PlatformArgs?.MotionEvent.ActionButton.ToString() ?? String.Empty)
```

Comment:
Should we remove this? If it's a TODO item, we could put it in a separate issue...

Reply:
I'd like to see if we can expand on this in the near future.  I just didn't want to get too crazy with the tests yet (especially since they're flaky on xharness)

Reply:
Removed comments